tests/e2e.test.js

```diff
@@ -295,17 +296,34 @@ describe('Ordering', () => {
     }
   });
 
-  it.skip('orders grid items by color', async () => {
-    // Items are ordered using the CSS property `order`...
-    await expect(page).toClick('#order-color');
+  it('orders grid items by color', async () => {
+    const $items = await page.$$('li.grid-item');
+    for (let i = 0; i < $items.length; i++) {
+      const $button = await $items[i].$('button.grid-item__color');
 
-    const $gridItems = await page.$$('.copy-color');
-    for (let i = 0; i < $gridItems.length; i++) {
-      const $gridItem = $gridItems[i];
-      const hex = hexes[i];
-      await expect($gridItem).toMatch(`#${hex}`);
+      const $text = await $items[i].evaluate((el) =>
+        JSON.stringify(el.getAttribute('style')),
+      );
+      const $idx = parseInt($text.substring(15, $text.length - 1));
+
+      const $color = await $button.evaluate((el) =>
+        JSON.stringify(el.textContent),
+      );
+
+      await expect($color).toMatch(`#${hexes[$idx]}`);
     }
   });
+
+  // issue: https://github.com/simple-icons/simple-icons-website/issues/66
+  it('compares the first two colors to make sure they are not black', async () => {
```

Comment:
Minor, this description doesn't follow the "style" of the rest of the tests. It describes what the tests does rather than what the test tests. A more "in style" description could be:

```suggestion
  it('doesn't have a black-like color at the start', async () => {
```

Reply:
no, you are right. Changing it.